README.md

```diff
@@ -0,0 +1,28 @@
+<h2>dev-ace: An AI Engine Based on LLaMA</h1>
```

Comment:
Please remove "dev-ace" from the Readme before it is merged into dev

Meta's own documentation mostly seems to use the capitalization "Llama" instead of "LLaMA" so I would like to do the same here.

The AI engine is not strictly based on Llama, rather it exists independent of Llama and will use Llama as an interface and allow for learning with, and modification of, nodes and edges based on their content.
A better summary would be: 
```
An AI system designed to exceed human-level problem-solving in general contexts by: 

1. encoding knowledge in a human-readable graph,
2. performing explicit and auditable reasoning,
3. incorporating new information via data ingestion algorithms assisted by LLMs, and
4. enhancing knowledge accuracy and predictive power through refinement algorithms.
```